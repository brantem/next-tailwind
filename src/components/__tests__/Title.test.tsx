import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

import Title from 'components/Title';

test('Title', () => {
  const { container } = render(<Title>Title</Title>);

  expect(container).toMatchSnapshot();
  expect(screen.queryByText('Title')).toBeInTheDocument();
});
