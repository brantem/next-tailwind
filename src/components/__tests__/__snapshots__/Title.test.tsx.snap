// Vitest Snapshot v1

exports[`Title 1`] = `
<div>
  <h1>
    Title
  </h1>
</div>
`;
